src/AccessibilityInsights/Modes/ConfigurationModeControl.xaml.cs

```diff
@@ -170,8 +179,10 @@ private async Task<bool> HandleReleaseSwitch()
                 ex.ReportException();
 
                 Dispatcher.Invoke(() =>
-                    MessageDialog.Show(string.Format(CultureInfo.InvariantCulture, Properties.Resources.ConfigurationModeControl_VersionSwitcherException))
-                );
+                {
+                    MainWin.Topmost = oldTopMost;
```

Comment:
Does oldTopMost need to be restored in the non-Exception case?

Reply:
The window will have closed and the function will have returned `true` at that point. If it hasn't, there should have been an exception thrown.

Reply:
Gotcha. The "return false" outside of the catch block made me think there might be a separate non-Exception path. (Stylistically, I'd probably put the return false inside the catch block, so that if another path were introduced, the compiler would catch it automatically)

---
In reply to: [278254819](https://github.com/Microsoft/accessibility-insights-windows/pull/337#discussion_r278254819) [](ancestors = 278254819)

Reply:
Moved that return!

Reply:
:thumbsup:

---
In reply to: [278312603](https://github.com/Microsoft/accessibility-insights-windows/pull/337#discussion_r278312603) [](ancestors = 278312603)